.gitlab-ci.yml

```diff
@@ -35,6 +35,7 @@ stages: # ---------------------------------------------------------------------
 .job_template: &template_hls-build
   <<: *template_base
   stage: hls-build
+  when: always
```

Comment:
does this belong here? why are we changing the CI on this PR?

Reply:
@pwittich Without this, the HLS build checks will not run if the code quality checks fail. Having the HLS build checks always run was useful at some point during debugging.

I figured this feature would probably be useful in future rounds of debugging, so I left it on by default. But I don't feel very strongly about this, and we can turn it off again, or move it to separate PR if you think that makes more sense.

Reply:
I would like to hear what @RobertGlein thinks, but he is out on parental leave. Maybe @aperloff has some thoughts?